bundles/AdminBundle/composer.json

```
@@ -0,0 +1,44 @@
+{
+  "name": "pimcore/admin-classic-bundle",
+  "license": "GPL-3.0+",
+  "type": "pimcore-bundle",
+  "config": {
+    "sort-packages": true,
+    "preferred-install": {
+      "pimcore/pimcore": "source",
+      "*": "dist"
+    }
+  },
+  "require": {
+    "pimcore/pimcore": "10.6.x-dev as 10.6 || ^11.0",
```

```suggestion
    "pimcore/pimcore": "^11.0",
```
Here we need `11.x` only, right?